src/lib/krb5/os/os-proto.h

```diff
@@ -83,6 +83,36 @@ struct sendto_callback_info {
     void *data;
 };
 
+/*
+ * Initialize with all zeros except for princ.  Set no_hostrealm to disable
+ * host-o-realm lookup, which ordinarily happens after canonicalizing the host
```

Comment:
"host-o-realm"?

Reply:
Typo for "host-to-realm", not actually a breakfast cereal.